openasip/opset/base/double.cc

```diff
@@ -134,7 +134,8 @@ END_OPERATION(DIVD)
 OPERATION(EQD)
 
 TRIGGER
-IO(3) = (!isnan(DBL(1)) && !isnan(DBL(2)) && DBL(1) == DBL(2)) ? 1 : 0;
+IO(3) = (!std::isnan(DBL(1)) && !std::isnan(DBL(2)) && DBL(1) == DBL(2)) ? 1
+                                                                         : 0;
```

Comment:
This still has the weird newline before the ":".

Reply:
Reverted from main, updated only the header